google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -115,6 +114,7 @@
 
 /** Implementation of Cloud Spanner remote calls using Gapic libraries. */
 public class GapicSpannerRpc implements SpannerRpc {
+  // Thread factory to use to create our worker threads
```

Comment:
minor - think you can remove this comment

Reply:
You're right, this is no longer valid.